### ISSUE TITLE
Electron-pdf issue in JSReport

### ISSUE THREAD

**Issue by virajitha-sarma** [bjrmatos/jsreport-electron-pdf#20]
Hi,
I am runnning jsreport in windows 10 and installed electron-pdf plugins in this way:
npm install jsreport-electron-pdf --save
npm install electron --save

I have attached jsreport.config.json and package,json configuration files

[jsreport.config.json.txt](https://github.com/bjrmatos/jsreport-electron-pdf/files/4535304/jsreport.config.json.txt)
[package.json.txt](https://github.com/bjrmatos/jsreport-electron-pdf/files/4535307/package.json.txt)

I am trying to generate pdf using electron-pdf engine using the reports api:
http://localhost:5488/reports/api
{
"template": {
"content" : "Hello world {{name}}",
"recipe": "electron-pdf",
"engine": "handlebars",
"chrome": {
"landscape": true
}
},
"data" : { "name": "JSreports PDF"},
}

**Getting this error during the processing of this  request**
2020-04-24T07:18:46.203Z - error: Error when processing render request 1 Worker Timeout, the worker process does not respond after 359127 ms Error: Worker Timeout, the worker process does not respond after 359127 ms
at Timeout._onTimeout (C:\jsreports\jsreportapp\node_modules\electron-workers\lib\ElectronManager.js:377:21)
at listOnTimeout (internal/timers.js:549:17)
at processTimers (internal/timers.js:492:7)

Please help me with this issue

**Comment by virajitha-sarma** [comment]
Also attached electron logs
[electronlogs.txt](https://github.com/bjrmatos/jsreport-electron-pdf/files/4535559/electronlogs.txt)

**Comment by bjrmatos** [comment]
the problem was explained here https://forum.jsreport.net/topic/1559/electron-pdf-issue-in-jsreport

TL;DR: an unsupported electron version was used
